src/docs/markdown/running.md

```diff
@@ -152,6 +156,20 @@ RestartSec=5s
 Then, save the file and exit the text editor, and restart the service for it to take effect:
 <pre><code class="cmd bash">sudo systemctl restart caddy</code></pre>
 
+### SELinux System Considerations
+
+On SELinux enabled systems you have two options:
+1. Install Caddy using the [COPR repo](https://copr.fedorainfracloud.org/coprs/g/caddy/caddy/). Your systemd file and caddy binary will be created and labelled correctly. If you wish to use a custom build of Caddy, you'll need to label the executable as described below.
+2. [Download Caddy from this site](https://caddyserver.com/download) or compile it with [`xcaddy`](https://github.com/caddyserver/xcaddy). In both cases you will need to label the files yourself.
+
+Systemd unit files and their executables will not be run unless labelled with `systemd_unit_file_t` and `bin_t` respectively.
+
+The `systemd_unit_file_t` is automatically applied to files created in `/etc/systemd/...`, so be sure to create your `caddy.service` file there.
+
+To tag the caddy binary, you can use the following commands:
+<pre><code class="cmd bash">semanage fcontext -a -t bin_t /usr/bin/caddy
+restorecon -Rv /usr/bin/caddy
+</code></pre>
```

Comment:
I think making it a one-liner is helpful here:
```suggestion
<pre><code class="cmd bash">semanage fcontext -a -t bin_t /usr/bin/caddy && restorecon -Rv /usr/bin/caddy
</code></pre>
```

I think it would render incorrectly on two lines (it would only show `$` on the first line with the CSS) unless we used a span for each line, like the below (if you don't like `&&`):

```suggestion
<pre><code class="cmd"><span class="bash">semanage fcontext -a -t bin_t /usr/bin/caddy</span>
<span class="bash">restorecon -Rv /usr/bin/caddy</span</code></pre>
```

Reply:
A oneliner is fine as long as it works ^^